extensions/podman/src/extension.spec.ts

```diff
@@ -1088,3 +1088,33 @@ test('checkDisguisedPodmanSocket: runs updateWarnings when called not on Linux',
   await checkDisguisedPodmanSocket(provider);
   expect(updateWarningsMock).toBeCalled();
 });
+
+test('Even with getJSONMachineList erroring, do not show setup notification on Linux', async () => {
+  vi.mocked(isLinux).mockReturnValue(true);
+  vi.spyOn(extensionApi.process, 'exec').mockRejectedValue({
+    name: 'name',
+    message: 'description',
+    stderr: 'error',
+  });
```

Comment:
I'm wondering if we need the spyOn as it's mocked probably vi.mocked(extensionApi.process.exec) would do the same trick ?

Reply:
unfortunately do need it, or else you get this error:

```
AssertionError: expected [Function] to throw error including 'description' but got 'wsl bootstrap script failed: exit sta…'

- Expected
+ Received

- description
+ wsl bootstrap script failed: exit status 0xffffffff
```

When not mocking it / using process.exec trick.